### PR TITLE
Traderjoe connector

### PR DESCRIPTION
**Pull request by crook-ed**

### CODE REVIEW

**Review by bhavik-m — COMMENTED** [review]
Why delete `.env.evample` file? No need to delete that

File: contracts/avalanche/connectors/traderjoe/main.sol [hunk 1]
```diff
@@ -0,0 +1,479 @@
...
+     * @param setId ID stores the amount of tokens deposited.
+    */
+    function deposit(
+        address token,
+        address jToken,
+        uint256 amt,
+        uint256 getId,
+        uint256 setId
+    ) external payable returns (string memory _eventName, bytes memory _eventParam) {
+        console.log(1);
```
Comment (bhavik-m) [inline contracts/avalanche/connectors/traderjoe/main.sol]:
```suggestion
```

**Comment by crook-ed** [comment]
> Why delete `.env.evample` file? No need to delete that

Yaah added it also.

**Comment by thrilok209** [comment]
Closing this for now. We can open this back when required

**Pull request closed without merging**